testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveRedirectionToIceberg.java

```diff
@@ -332,6 +332,26 @@ public void testAlterTableAddColumn()
         onTrino().executeQuery("DROP TABLE " + icebergTableName);
     }
 
+    @Test(groups = {HIVE_ICEBERG_REDIRECTIONS, PROFILE_SPECIFIC_TESTS})
+    public void testAlterTableDropColumn()
+    {
+        String tableName = "iceberg_alter_table_drop_column_" + randomTableSuffix();
+        String hiveTableName = "hive.default." + tableName;
+        String icebergTableName = "iceberg.default." + tableName;
+
+        createIcebergTable(icebergTableName, false);
+
+        onTrino().executeQuery("ALTER TABLE " + hiveTableName + " DROP COLUMN comment");
+
+        Assertions.assertThat(onTrino().executeQuery("DESCRIBE " + icebergTableName).column(1))
+                .containsOnly("nationkey", "name", "regionkey");
+
+        assertResultsEqual(
```

Comment:
what is the purpose of this assertion?

Reply:
The assertion verifies that there was no rows inserted/deleted during the `DROP COLUMN` statement.

Reply:
okay, seems a bit defensive here (as opposed to `testDelete`), but let's keep it.